### DISCUSSION TITLE
How do we Register the class in Rule Engine for creating custom code?

### DISCUSSION THREAD

**Discussion by sugeshshun** [microsoft/RulesEngine#434] [Q&A]
We are trying to create custom method in rule engine. As per the document we need to register the class (Utils) to call the custom method

I have tried like below, but am not sure my class is registered or not? 
"Expression": "Utils.CheckContains(input1.country, \"india,usa,canada,France\") == true"

**Comment by peeveen** [comment]
It's described [here](https://github.com/microsoft/RulesEngine/wiki/Getting-Started#resettings).
When you create the `RulesEngine`, pass in a `ReSettings` object containing a `CustomTypes` definition that contains your Utils class type. You can then use the methods in that class in your expressions.

```c#
var reSettingsWithCustomTypes = new ReSettings { CustomTypes = new Type[] { typeof(Utils) } };
new RulesEngine.RulesEngine(workflowRules.ToArray(), null, reSettingsWithCustomTypes);
```
